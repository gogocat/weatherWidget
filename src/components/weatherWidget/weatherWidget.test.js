import React from 'react';
import { shallow } from 'enzyme';
import WeatherWidget from './weatherWidget';

const mockGeolocation = {
    getCurrentPosition: jest.fn()
      .mockImplementationOnce((success) => Promise.resolve(success({
        coords: {
          latitude: 51.1,
          longitude: 45.3
        }
      })))
  };

global.navigator = { geolocation: mockGeolocation };

it('renders without crashing', () => {
    
    shallow(<WeatherWidget />);
});