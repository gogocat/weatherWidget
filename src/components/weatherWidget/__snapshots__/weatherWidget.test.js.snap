// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <Context.Consumer>
    [Function]
  </Context.Consumer>,
]
`;
